File: ACBrFramework.Net/ACBrFramework.Net/PAF/Registros/PAF_A/CodigoTipoDocumento.cs
#region COM_INTEROP
#if COM_INTEROP

using System.Runtime.InteropServices;

#endif
#endregion COM_INTEROP

using System.ComponentModel;
namespace ACBrFramework.PAF
{
    #region COM_INTEROP
#if COM_INTEROP
	[ComVisible(true)]
#endif
    #endregion COM_INTEROP

    public enum CodigoTipoDocumento
    {
        /// <summary>
        /// Cupom Fiscal
        /// </summary>
        [Description("Cupom Fiscal")]
        CupomFiscal = 1,

        /// <summary>
        /// Comprovante Não Fiscal
        /// </summary>
        [Description("Comprovante Não Fiscal")]
        ComprovanteNaoFiscal = 2,

        /// <summary>
        /// Nota Fiscal
        /// </summary>
        [Description("Nota Fiscal")]
        NotaFiscal = 3
    }
}

File: ACBrFramework/jACBrFramework/jACBrFramework/src/jACBrFramework/paf/CodigoTipoDocumento.java
package jACBrFramework.paf;

/**
 * Codigo do tipo de documento.
 * 
 * @author Jose Mauro
 * @version Criado em: 10/12/2013 15:54:59, revisao: $Id$
 */
public enum CodigoTipoDocumento {
    // <editor-fold defaultstate="collapsed" desc="Enums">    
    CUPOM_FISCAL("Cupom Fiscal", "1"),
    COMPROVANTE_NAO_FISCAL("Comprovante Não Fiscal", "2"),
    NotaFiscal("Nota Fiscal", "3");
    // <editor-fold defaultstate="collapsed" desc="Attributes">    
    /**
     * Descricao do tipo de documento.
     */
    private String descricao;
    /**
     * Codigo do tipo de documento.
     */
    private String codigo;
    // </editor-fold>
    // <editor-fold defaultstate="collapsed" desc="Constructors">  
    CodigoTipoDocumento(String pDescricao, String pCodigo) {
        descricao = pDescricao;
        codigo = pCodigo;
    }
    // </editor-fold>
    // <editor-fold defaultstate="collapsed" desc="Methods">    
    /**
     * Descricao do tipo de documento.
     * 
     * @return the descricao
     */
    public String getDescricao() {
        return descricao;
    }

    /**
     * Codigo do tipo de documento.
     * 
     * @return the codigo
     */
    public String getCodigo() {
        return codigo;
    }
    // </editor-fold>    
}